net/browserless_agent/tests/test_browserless_agent.ts

```diff
@@ -158,8 +170,29 @@ test("test browserless error response", async () => {
       false,
     );
 
-    const resultData = (result as any).error.result;
-    assert.ok(resultData.onError, "Expected error result");
+    const resultData = result.error?.result;
+    assert.ok(resultData?.onError, "Expected error result");
+  } finally {
+    cleanupEnvironment();
+  }
+});
+
+test("test browserless api key from env", async () => {
+  setupEnvironment();
+  try {
+    const result = await graphDataTestRunner<{
+      result: string;
+    }>(
+      __dirname,
+      __filename,
+      graphDataApiKeyFromEnv,
+      { browserlessAgent: browserlessAgentInfo, copyAgent } as any,
```

Comment:
Here, the 'any' type is used to handle the type error. There seems to be a type incompatibility between 'copyAgent' and 'graphDataTestRunner'.

<img width="1252" alt="image" src="https://github.com/user-attachments/assets/8da4cb13-b47d-4ff4-827d-8698e3bd5dc9" />

Reply:
Thank you!!
I'll check this and fix it!